policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_exemptions.py

```diff
@@ -9,3 +9,5 @@ class vt_exemptions(Variable):
     definition_period = YEAR
     reference = "https://www.mainelegislature.org/legis/statutes/36/title36sec5126-A.html"
     defined_for = StateCode.VT
+
+    adds = ["vt_personal_exemptions"]
```

Comment:
Is this because it includes other exemption types?

If so please make placeholders for those.

Reply:
Hi Max. I think the personal exemptions is the only type of exemptions we have when calculating Vermont taxable income. Are there additional changes I should make if this is the case? Thanks. @MaxGhenis

Reply:
We don't need this redundant variable then. Just pull in `vt_personal_exemptions` where you would have used `vt_exemptions`.

Reply:
Sure thing.